admin/services/check-form-v2.service.js

```diff
@@ -43,6 +43,9 @@ checkFormV2Service.prepareSubmissionData = async (uploadedFiles, checkFormType)
     return new Promise((resolve, reject) => {
       csv.fromPath(uploadedFile.file, { headers: false, trim: true })
         .on('data', function (row) {
+          if (!row[0] && !row[1]) {
```

Comment:
@jon-shipley @GuyHarwood It was agreed that we can go for a more tolerant approach on the check form submission rule about blank lines.
I reverted the trim removal change from the validation layer and added this change to discard lines with empty content(blank)

Reply:
lovely 🏆

Reply:
@GuyHarwood this is going to trigger validation errors.
Mid-file blank lines are not allowed.
trim will only remove blank lines from beginning and end of the file.